src/views/Tickets/components/TicketsCard/TicketsCard.js

```diff
@@ -68,73 +68,77 @@ const TicketsCard = ({ data, isTrip }) => {
   };
 
   return (
-    <Card className={classes.root}>
-      <CardHeader
-        avatar={
-          <Avatar aria-label="recipe" className={classes.avatar}>
-            {`${content.firstName[0]}${content.lastName[0]}`}
-          </Avatar>
-        }
-        action={<CloseEditButton id={id} />}
-        title={`${content.firstName} ${content.lastName}`}
-        subheader={new Date().toLocaleString()}
-      />
+    <>
+    { !ticket.closed &&
```

Comment:
Before adding this line, there was error while navigating to `/support` after fetching tickets in `LatestSupportTickets` component (`dashboard`): 
![image](https://user-images.githubusercontent.com/44790071/82249997-02fdb600-994b-11ea-8246-a36f46c4b726.png)

Error was thrown becouse tickets with closed status have `content` property equals to: `tickets.content === 'Close ticket'`